release-notes/v1_75.md

```diff
@@ -264,7 +264,7 @@ There is a new setting `terminal.integrated.tabStopWidth` that configures the ta
 
 ### Powerline triangle and diagonal line custom glyphs
 
-GPU-accelerated terminals now get pixel-perfect custom glyphs for the triangle and diagonal line powerline extra symbols glyphs. These characters are somewhat ambiguous whether they are single or double which charatcers and differ depending on the font used, we chose to render them as single width.
+GPU-accelerated terminals now get pixel-perfect custom glyphs for the triangle and diagonal line powerline extra symbols glyphs (`U+E0B8-U+E0BF`). These characters are somewhat ambiguous whether they are single or double which characters and differ depending on the font used, we chose to render them as single width.
```

Comment:
Should this be "single or double width characters"?

Reply:
Yes it should, fixed in vnext https://github.com/microsoft/vscode-docs/commit/29082268f38f3a7cce10100fb4b018cc5ea35774